featurex/extractors/image.py

```diff
@@ -9,6 +9,7 @@
 import numpy as np
 import tempfile
 import os
+import pySaliencyMap
```

Comment:
a note while we're figuring out the license issue: pySaliencyMap is not set up as a package in its current implementation.